Use OCI Transport for embeded images to save time

rhatdan opened the pull request:
No reason to use containers/storage for instructlab or vllm since we are only building for embedding within a bootc image. By storing directly in OCI, we can save many minutes and lots of disk size.

n1hility commented:
LGTM

lmilbaum reviewed (approved):
LGTM. Would be useful if the work @Gregory-Pereira and I are doing can validate this change.

rhatdan committed:
Use OCI Transport for embeded images to save time

No reason to use containers/storage for instructlab or vllm
since we are only building for embedding within a bootc image.
By storing directly in OCI, we can save many minutes and lots of
disk size.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

sallyom commented:
LGTM, also - as long as it works (I haven't tested this)!
